SwSpotify/spotify.py

```diff
@@ -23,7 +28,13 @@ def find_spotify_uwp(hwnd, windows):
         text = win32gui.GetWindowText(hwnd)
         classname = win32gui.GetClassName(hwnd)
         if classname == "Chrome_WidgetWin_0" and len(text) > 0:
-            windows.append(text)
+            _, proc_id = win32process.GetWindowThreadProcessId(hwnd)
```

Comment:
looks good as long as it works, but we should document it better so it's easier to maintain since currently it looks like windows magic is happening and then we get the path.